<ui version="4.0" >
 <class>PropertyAutoClassifyFormClass</class>
 <widget class="GUIFormBase" name="PropertyAutoClassifyFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PropertyAutoClassifyForm</string>
  </property>
  <widget class="QLabel" name="label_30" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>登録されているライブラリリスト</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibList" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior" >
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text" >
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonEditLibrary" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>ライブラリ編集</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPickupTest" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>254</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>単一ライブラリでの抽出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEditLibFolder" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>分類選択</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetFromAll" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>170</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>=>></string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGetBack" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>120</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>50</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>今回使用する設定</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetFrom" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>90</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>=></string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLibFolderName" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>ＸＸＸ</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetGeneratedLibList" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>70</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionBehavior" >
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text" >
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ButtonGenerateLibs" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>360</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>上の全てのライブラリで生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGetBackAll" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>200</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;&lt;=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPickupClearTest" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>抽出のクリア</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>250</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>ライブラリプロパティ</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditNoiseSize" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditSpaceToOutline" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_16" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>優先度</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditPriority" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_14" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>輪郭からの間隔</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_15" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>ノイズサイズ</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonLibSave" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text" >
     <string>データベースへの更新</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>450</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QTableWidget" name="tableWidgetBlockInfo" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <column>
     <property name="text" >
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>ライブラリ名</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>領域総数</string>
     </property>
    </column>
   </widget>
   <widget class="QToolButton" name="toolButtonAreaMode" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>生成された領域情報</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
